### ISSUE TITLE
GRNsight upload should count exactly once per upload

### ISSUE THREAD

**Issue by kdahlquist** [dondi/GRNsight#196]
Right now, Google Analytics does not appear report multiple loads during the same browser session. This is good to avoid inflating web site results, but is not what we want for uploads. This behavior should be changed.

**Comment by dondi** [comment]
Oddly enough, in my testing, I did get exactly one upload “bump” for every upload that I performed. It is possible that the non-repetition only occurs in the public dondi.github.io version of the site.

Nevertheless, I found a tweak in the [Google Analytics Developer Guides](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference) that might enforce this non-repetition, and I have pushed it to beta. We will have to deploy this version and perform some tests on the dondi.github.io site to see if this tweak genuinely works, because as mentioned the local version seems to count uploads correctly.
